src/hbbft.erl

```diff
@@ -22,6 +23,7 @@
           j :: non_neg_integer(),
           round = 0 :: non_neg_integer(),
           buf = [] :: [binary()],
+          max_buf = infinity :: infinity,
```

Comment:
We need a way to set this, right?

Reply:
right, will add to init for the time being.